infra/cifuzz/actions/run_fuzzers/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+# Docker image to run CIFuzz run fuzzers action in.
+
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install -y git \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    gnupg2 \
+    software-properties-common \
+    python3
+
+
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - && apt-key fingerprint 0EBFCD88
+RUN add-apt-repository \
+   "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+   xenial \
+   stable"
+
+RUN apt-get update && apt-get install docker-ce docker-ce-cli containerd.io -y
+
+ENV OSS_FUZZ_ROOT=/opt/oss-fuzz
+RUN git clone https://github.com/google/oss-fuzz.git ${OSS_FUZZ_ROOT}
+
+# Copies your code file from  action repository to the container
+COPY run_fuzzers_entrypoint.py /opt/run_fuzzers_entrypoint.py
```

Comment:
The rest of the file is the same as `build_fuzzers` one. Are all these deps needed for running the fuzzers?
- If no, let's remove them and keep the image smaller / faster
- If yes, let's put those in one place and inherit from it

Reply:
The answer is yes but there is a difficulty with inheriting from docker images locally with actions. When an action is called, just the repo that contains the action is accessible so we could not access a parent directory containing a parent image. I think a propper workaround for this is to upload this base image to Jenkins and just use that as the base image. What do you think?